sdks/python/apache_beam/runners/interactive/extensions/interactive_beam_side_panel/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Why these options?

Also do we need the license header here?

Reply:
These are default options used by the jupyter labextension template. They direct how the TypeScript is compiled.

The license header is needed to publish this package to npm. So moving this whole package into a separate repo might be a better option.